src/common/snippets/src/op/subgraph.cpp

```diff
@@ -72,10 +72,14 @@ void Subgraph::set_virtual_port_count(const size_t count) {
 
 void Subgraph::set_min_jit_work_amount(const size_t jit_work_amount) {
     config.m_min_jit_work_amount = jit_work_amount;
+    if (m_linear_ir)
+        m_linear_ir->set_min_kernel_work_amount(jit_work_amount);
```

Comment:
I'm not sure that it's good idea. Because user can call this method before LinearIR creation. Thus, this value is not propagate to LinearIR.
For sure, we can add comment aka `the method should be called after LinearIR creation but before control_flow transformations`. But it's not safely for me at least. What do you think?

Reply:
@IvanNovoselov updated the changes after offline discussion. Could you take a look please one more time?